definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -0,0 +1,9 @@
+[
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "IssueAOSOffline_Respondent_fromAosAwaiting",
```

Comment:
@yarpo, as per our conversation, in this specific case, there's no need to have this event as "R" in prod. We only did that because we didn't have feature-toggle.

Reply:
Would you be able to elaborate on this?